src/auto-check-element.ts

```diff
@@ -176,6 +176,10 @@ export class AutoCheckElement extends HTMLElement {
   set csrfField(value: string) {
     this.setAttribute('csrf-field', value)
   }
+
+  get httpMethod(): string {
+    return this.getAttribute('http-method') || 'POST'
```

Comment:
This should probably just return allowed values, eg just GET or POST. Otherwise this allows for arbotrary verbs. Might lead to undesired behaviour. 

We can ship this and bugfix later if time is of the essence.

Reply:
WDYT of 059dde3?